packages/@lightningjs/ui-components/src/docs/ThemingMode.mdx

```diff
@@ -36,7 +36,7 @@ By default each component will attempt to find a `focused` mode when thte compon
 
 ### Theme Level
 
-You can set any component's default mode by specifying it in the `componentConfig` property of your theme. For a deeper dive on `tones`, see ["Component Config"](../?path=/docs/docs-theming-component-config--page).
+You can set any component's default mode by specifying it in the `componentConfig` property of your theme. For a deeper dive on `tones`, see ["Component Config"](?path=/docs/docs-theming-component-config--docs).
```

Comment:
should this be pointing to a deeper dive on modes rather than tones?
```suggestion
You can set any component's default mode by specifying it in the `componentConfig` property of your theme. For a deeper dive on `modes`, see ["Component Config"](?path=/docs/docs-theming-component-config--docs).
```